Gemfile.lock

```diff
@@ -0,0 +1,147 @@
+PATH
```

Comment:
Gems should avoid checking in the `Gemfile.lock` otherwise other packages that use this library will be bound to these versions.

Reply:
Sure I can add it to .gitignore